packages/expo-ui/ios/GridView.swift

```diff
@@ -0,0 +1,51 @@
+import ExpoModulesCore
+import SwiftUI
+
+// MARK: - Grid Props
+final class GridProps: ExpoSwiftUI.ViewProps, CommonViewModifierProps {
```

Comment:
We recently migrated from `CommonViewModifierProps` to `UIBaseViewProps` API. Should be good to merge once we do the same here. 
example: https://github.com/expo/expo/blob/5d3f4eac729d9ee0206202e138416587e6669a88/packages/expo-ui/ios/EllipseView.swift#L6